src/runtime_src/core/tools/xbutil2/SubCmdValidate.cpp

```diff
@@ -1349,9 +1349,21 @@ get_platform_info(const std::shared_ptr<xrt_core::device>& device,
 {
   auto bdf = xrt_core::device_query<xrt_core::query::pcie_bdf>(device);
   ptTree.put("device_id", xrt_core::query::pcie_bdf::to_string(bdf));
-  ptTree.put("platform", xrt_core::device_query<xrt_core::query::rom_vbnv>(device));
-  ptTree.put("sc_version", xrt_core::device_query<xrt_core::query::xmc_sc_version>(device));
-  ptTree.put("platform_id", (boost::format("0x%x") % xrt_core::device_query<xrt_core::query::rom_time_since_epoch>(device)));
+
+  boost::property_tree::ptree platform_report;
+  boost::property_tree::ptree empty_ptree;
+  auto report = std::make_shared<ReportPlatforms>();
+  report->getPropertyTreeInternal(device.get(), platform_report);
+  
+  const boost::property_tree::ptree& platforms = platform_report.get_child("platforms", empty_ptree);
+  for(auto& kp : platforms) {
```

Comment:
Is it possible for there to be more than 1 platform in the collection.   

If it is then the data below will overwrite the previous data...which isn't good.

If it isn't, then it would be nice to add a check right before the for loop to see if the number of elements is 0 or 1.  If not then and internal exception should be raised.

Reply:
It isn't possible to have more than one platform. (At least not yet)